kdump-lib-initramfs.sh

```diff
@@ -102,8 +102,15 @@ get_fs_type_from_target()
 
 get_mntpoint_from_target()
 {
-	# --source is applied to ensure non-bind mount is returned
-	get_mount_info TARGET source "$1" -f
+	local SOURCE TARGET
+	findmnt -k --pairs -o SOURCE,TARGET "$1" | while read line; do
+		eval "$line"
+		# omit sources that are bind mounts i.e. they contain a [/path/to/subpath].
+		if [[ ! "$SOURCE" =~ \[ ]]; then
+			echo $TARGET
+			break
+		fi
+	done
```

Comment:
I really like to avoid 'eval' here. Especially as it's not necessary.
For me the following worked
```
while read _source _target; do
    ...
done < <(findmnt -k -n -o SOURCE,TARGET "$1")
```
Other than that the series looks fine to me.

Reply:
That would fail if the source or target contained whitespace, which is pretty unlikely of course...but, still.

(OTOH, the real fix is to not write any nontrivial shell script at all...)

Reply:
Hi @prudo1 , I think we can merge this first since it works well now. And we can make changes later to avoid code conflicts if we touch this code in the future.